dss/stepfunctions/visitation/storage.py

```diff
@@ -81,6 +96,21 @@ def walker_walk(self) -> None:
                     else:
                         logger.debug(f"Replica {replica} contains {key}")
                         self.work_result['present'][i] += 1
+                        if values is not None:
+                            values[folder.inspect_key(i, handles[i], key)].add(replica)
+                if values is not None:
+                    assert len(values) > 0
+                    if len(values) > 1:
+                        logger.warning('Inconsistency detected for key %s: %r', row.min, values)
```

Comment:
It might be useful to include the Step Functions execution name in the log messages, which would serve as a correlation id.

Reply:
I agree, but this should be done for all visitations consistently.